docs/cristin.ttl

```diff
@@ -476,8 +476,9 @@
 :TYPE_BOK_RAPPORT
     a               :cristinfield ;
     rdfs:comment    """
-    A type of book called rapport.
-    """ ;
+        Metadata about a book.
+        This only has a value if the varbeidhovedkatkode has one of the following values OVERSETTELSESARB RAPPORT BOK
+        """ ;
     rdfs:range      :TYPE_BOK_RAPPORT_Class ;
```

Comment:
The class should also be defined. See towards top of the file for examples.
Suggestion:
```

:TYPE_BOK_RAPPORT_Class
 a :CristinClass;
         rdfs:comment """
         The respetive object contains metadata about a book. The Cristin secondary categories (varbeidunderkatkode)
         that are expected to have such an entry are the following:
         ANTOLOGI
         DRGRADAVH
         FAGBOK
         FORSKERLINJEOPPG
         HOVEDFAGSOPPGAVE
         KOMMENTARUTG
         KOMPENDIUM
         LEKSIKON
         LISENSIATAVH
         LÆREBOK
         MAGISTERAVH
         MASTERGRADSOPPG
         MONOGRAFI
         OPPSLAGSVERK
         OVERSETTELSE_BOK
         POPVIT_BOK
         RAPPORT
         UTSTILLINGSKAT
         """
```